adapter/internal/oasparser/envoyconf/routes_with_clusters_test.go

```diff
@@ -214,6 +216,25 @@ func createDefaultBackendRef(serviceName string, port int32, weight int32) gwapi
 	}
 }
 
+func getBackendProperties(httpRoute *gwapiv1b1.HTTPRoute) dpv1alpha1.BackendPropertyMapping {
```

Comment:
why are we duplicating util functions just for tests, isn't it the point of testing is to test existing util functions?

Reply:
moved part of test util to default util and used in  both test and actual code